spec/config.cr

```diff
@@ -25,24 +25,34 @@ require "../src/jennifer"
 def set_default_configuration
   Jennifer::Config.reset_config
   Jennifer::Config.configure do |conf|
-    # conf.logger.level = Logger::DEBUG
-    conf.logger.level = Logger::ERROR
-    conf.host = "localhost"
-    conf.adapter = Spec.adapter
-    conf.migration_files_path = "./examples/migrations"
-    conf.db = "jennifer_test"
+    apply_config(conf)
+  end
+  Jennifer::Config.configure(:other_config) do |conf|
```

Comment:
This is annoying for this to need to be here, but because of how the models get initialized, when a model that uses this other config gets loaded it expects the config to be available so it needs to be created early. Right now its just a copy of the other config so that database can still be connected to during test.